_posts/2023-10-20-anvio-org-tutorial.md

```diff
@@ -0,0 +1,168 @@
+
+```bash
+bundle exec jekyll serve --incremental --trace
+```
+
+The `--incremental` flag ensures that the page will be re-rendered every time Jekyll detects a change in the file, so you can edit and see those changes immediately just by refreshing the webpage. The `--trace` flag makes sure you will see error tracebacks.
+
+If you post is dated in the future, you will need to tell Jekyll to include it in the local site by including the `--future` flag (since future posts are not rendered by default):
+
+```bash
+bundle exec jekyll serve --incremental --trace --future
+```
+
+The Jekyll output should tell you the local URL of anvio.org. For instance, on my machine, it was `http://127.0.0.1:4000`. You open that in a web browser and navigate through the website yourself to find your post, or you can append the `url` you specified earlier (in the YAML file) to the end of that URL and copy-paste that to your web browser to get to the post's webpage directly. For instance, while writing this post, I viewed it at the local URL [http://127.0.0.1:4000/docs/anvio-org-tutorial/](http://127.0.0.1:4000/docs/anvio-org-tutorial/).
+
+In addition to checking that your post renders nicely, you should also make sure you can see it featured under the appropriate section on the `Learn` page. If you don't see it there, or the link to your post is broken, please make sure that the URLs in the YAML file and the front matter match. If they do and you still don't see it; you should try deleting the existing local copy of the website and re-generating it from scratch, since sometimes Jekyll doesn't pick up on changes to our resource files.
+
+```bash
+rm -rf _site/; bundle exec jekyll serve --incremental --trace --future
+```
+
+{:.notice}
+Every file under the `_posts` directory, even if it is not a blog post, should appear on our 'Community Blog' page (locally rendered at [http://127.0.0.1:4000/blog/](http://127.0.0.1:4000/blog/)). In contrast, the 'Learn' page (locally rendered at [http://127.0.0.1:4000/learn/](http://127.0.0.1:4000/learn/)) can include links to external URLs and organizes the posts by section according to which YAML file they were described in. 
+
+### Adding images
+
+If your post includes images, you should add those files under the `images/` folder of the anvio.org repository. Our convention is to create a folder for images specific to that post within that directory, as in `images/anvio-org-tutorial/`, and add all associated image files there. Then, see [this web tip](https://merenlab.org/web-tips/#images) for instructions on how to include the image(s) within the markdown file for your post. Don't forget to commit your images to the repository, and ensure that they render properly.
+
+## Step 8) Make a PR
+
+Once you are done with your post, you should create a Pull Request (PR) on GitHub. After we review the PR and merge it into the main branch of the anvio.org repository, your 
```

Comment:
WELL, YOUR WHAT? 😂

Reply:
OMG 😂 so embarrassing. I will pretend that I did it on purpose just so people can see your brilliant example of a comprehensive PR review